import/import_script.go

```diff
@@ -33,7 +35,7 @@ func main() {
 	log.Println("Terraform Import Script Starting")
 
 	// collect all arguments from the user, indicating all the resources that need to be imported
-	resourcesPtr := flag.String("resources", "", "Comma separated list of resources to import. The only accepted values are workspace, deployment, cluster, api_token, team, team_roles, user_roles")
+	resourcesPtr := flag.String("resources", "workspace,deployment,cluster,api_token,team,team_roles,user_roles", "Comma separated list of resources to import. The only accepted values are workspace, deployment, cluster, api_token, team, team_roles, user_roles")
```

Comment:
added default value

when users run `./import_script` even when `-resource` flag is not set we will automatically default to importing all resources - is this ok?

Reply:
Yep